duckduckgo/BrowserServicesKit
# remove www dropping from suggestions (macOS)

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL: https://app.asana.com/0/230839424936/1204396770411787/f
iOS PR: https://github.com/duckduckgo/iOS/pull/1889
macOS PR: https://github.com/duckduckgo/macos-browser/pull/1421
What kind of version bump will this require?: Patch

**Description**: Removes www dropping from suggestions



**Steps to test this PR**:
1. on macOS on the search bar type softether you should get a softether.org suggestion (with the globe) click on it and check it loads the website.
1. on iOS on the search bar type softether you should get a www.softether.org suggestion (with the globe) click on it and check it loads the website.



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
